### PR TITLE
Config for GoMetricsWorker

### PR DESCRIPTION
**Pull request by justinvdm**

### CODE REVIEW

**Comment by justinvdm** [comment]
Ready for review.

File: go/vumitools/metrics_worker.py [hunk 1]
```diff
@@ -4,12 +4,21 @@
 from twisted.internet.task import LoopingCall
 
 from vumi import log
-from vumi.service import Worker
+from vumi.worker import BaseWorker
+from vumi.config import ConfigInt
 
 from go.vumitools.api import VumiApi, VumiApiCommand
+from go.vumitools.app_worker import GoWorkerConfigMixin, GoWorkerMixin
```
Comment (justinvdm) [inline go/vumitools/metrics_worker.py:11]: It feels a bit wierd importing things from `app_worker`. We seem to have base worker and base router worker infrastructure in there, so maybe we should change its name to `worker`? Not sure if that should be done in this PR.

Comment (jerith) [inline go/vumitools/metrics_worker.py:11]: This is one of those places that needs a bunch of cleanup, but we don't have to do that now.

File: go/vumitools/metrics_worker.py [hunk 2]
```diff
@@ -4,12 +4,21 @@
...
 
 
-class GoMetricsWorker(Worker):
+class GoMetricsWorkerConfig(BaseWorker.CONFIG_CLASS, GoWorkerConfigMixin):
+    metrics_interval = ConfigInt(
+        "How often the worker should send 'collect_metric' commands",
+        default=300,
+        static=True)
```
Comment (jerith) [inline go/vumitools/metrics_worker.py:19]: You should probably mention that this is in seconds.

Comment (justinvdm) [inline go/vumitools/metrics_worker.py:19]: Good idea

File: go/vumitools/metrics_worker.py (hunk 1 above)
Comment (justinvdm) [inline go/vumitools/metrics_worker.py:11]: Ok cool.

**Comment by jerith** [comment]
:+1: after the config help string update.

**Comment by justinvdm** [comment]
Ready for another review.

**Comment by jerith** [comment]
:+1:

**Pull request merged**